### PR TITLE
Feature: Add docs directory

### LINKED ISSUE #2: Create documentation
**Description**

We need to create some docs to people know what is this project for or how can it be used.

**Acceptance criteria**

- [ ] Create a docs folder
- [ ] Add needed files to create github pages (See: https://github.com/halivert/maths/tree/main/docs)
- [ ] Explain how can we use this repo with an example

### PR DESCRIPTION
**Pull request by Markaeus**
## Description
Added documentation folder with corresponding necessary github page files as well as matrix topics for its use.

Issue Resolves #2 

## Approach
I wasn't sure what exactly was meant for give an example. So I created several topics in which matrices were used in mathematics. I also noticed the documentation was in spanish for your example files. I'm a little rusty but tried my best to align with your example. Please let me know if I need to fix anything!

## Acceptance criteria

- [x] Create a docs folder
- [x] Add needed files to create github pages (See: https://github.com/halivert/maths/tree/main/docs)
- [ ] Explain how can we use this repo with an example

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+remote_theme: halivert/halivert.github.io@main
+
+title: "Matemáticas"
```
Comment (halivert) [inline docs/_config.yml]: Please rename it to: "Matrices"

File: docs/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- BigNum features:
+  - Two constructors (long long, string).
+  - Operator overload.
+  - int and long long conversion.
+  - Basic operations:
+    - Division.
+    - Multiplication.
+    - Modulus.
+    - Addition.
+    - Subtraction.
```
Comment (halivert) [inline docs/release-notes.md]: This file is not needed, as repo hasn't a release yet.

**Review by halivert — CHANGES_REQUESTED** [review]

File: docs/_config.yml (hunk 1 above)
Comment (Markaeus) [reply docs/_config.yml]: Done

File: docs/release-notes.md (hunk 2 above)
Comment (Markaeus) [reply docs/release-notes.md]: Done

**Review by halivert — APPROVED** [review]
Awesome, thank you!

**Pull request merged**